common/tools/dev-tool/src/commands/test-proxy/index.ts

```diff
@@ -5,11 +5,12 @@ import { subCommand, makeCommandInfo } from "../../framework/command";
 
 export const commandInfo = makeCommandInfo(
   "test-proxy",
-  "runs the proxy-tool with the `docker run ...` command"
+  "runs the proxy-tool using the .NET standalone executable"
```

Comment:
would it still expect the dependencies such as C++ executables, etc expected by the dotnet cli/app?

Reply:
Nope, nothing.

Reply:
Lovely